### PR TITLE
[docker]:Add docker build way.

### PR DESCRIPTION
**Pull request by Whojohn**
Problem Summary:

### What is changed and how it works?

Proposal: Add docker build way to quick start.

What's Changed:
1. Add quick start demo on docker.

How it Works:
1. Build special version by docker compose.

### CODE REVIEW

**Review by Al-assad — APPROVED** [review]
LGTM

**Comment by wolfboys** [comment]
good job.

File: docker/streamx/start.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+WORK_SPACE="/opt/streamx/bin"
+if [ ! -d $WORK_SPACE ]; then
+ tar xf /tmp/streamx-console-service-${STREAMX_VERSION}-bin.tar.gz -C /opt  \
+ && cp -R /opt/streamx-console-service-${STREAMX_VERSION}/* /opt/streamx     \
+ && mkdir /opt/jdk  \
+ && tar xf /tmp/jdk-8.tar.gz -C /opt/jdk --strip-components 1  \
+ && sed -i 's/localhost:3306/streamx-mysql-docker:3306/g' /opt/streamx/conf/application.yml \
+ && rm -rf /tmp/* \
+ && rm -rf /opt/streamx-console-service-${STREAMX_VERSION} \
+ && echo "Ini project success"
```
Comment (BruceWong96) [inline docker/streamx/start.sh]: “Initialize the project successfully.” might be better.

**Comment by wolfboys** [comment]
@Whojohn  The PR need to speed up the progress 💪🏻

**Comment by Whojohn** [comment]
> @Whojohn The PR need to speed up the progress 💪🏻

**Sorry， Too busy to handle my life ,**
I forgot it  , i will finish all job before 2022.2.27.  

- which include:
1. Docker quick start (All in one dockerfile. Not everyone need dockercompose to split mysql and streamx).
2. Prebuild docker image in china.net  which  improve experience .

**Pull request closed without merging**